src/scss/_variables.scss

```diff
@@ -91,12 +91,24 @@
 //
```

Comment:
Instead of modifying the bootstrap _variables.scss would it make sense to create a _ucf_variables.scss included after the _variables.scss and overriding the bootstrap variables? This would make updating bootstrap easier especially since it is still alpha.